### ISSUE TITLE
Compilation errors on rustc 1.63

### ISSUE THREAD

**Issue by antekone** [suzusuzu/virtualization-rs#1]
I'm trying to compile virtualization-rs using rustc 1.63, but I'm getting those errors:

```
$ cargo build
Compiling libc v0.2.132
Compiling block v0.1.6
Compiling malloc_buf v0.0.6
Compiling objc v0.2.7
Compiling virtualization-rs v0.1.2 (/Users/antek/dev/thirdparty/virtualization-rs)
error[E0054]: cannot cast as `bool`
--> src/base.rs:117:13
|
117 |             obj as BOOL
|             ^^^^^^^^^^^ unsupported cast

error[E0054]: cannot cast as `bool`
--> src/virtualization/virtual_machine.rs:196:20
|
196 |                 Ok(obj as BOOL)
|                    ^^^^^^^^^^^ unsupported cast

error[E0308]: mismatched types
--> src/virtualization/virtual_machine.rs:255:23
|
255 |             Ok(ret == 1i8)
|                       ^^^ expected `bool`, found `i8`

Some errors have detailed explanations: E0054, E0308.
For more information about an error, try `rustc --explain E0054`.
error: could not compile `virtualization-rs` due to 3 previous errors
FAIL: 101
```

Did anyone have a similar issue? Is there any quick fix for this?

**Comment by suzusuzu** [comment]
Thanks for reporting the issue. I will try to fix it, so please wait a while.

**Comment by lpicanco** [comment]
Opened a PR with a fix(#2)

**Comment by suzusuzu** [comment]
@antekone 
Sorry, I was disappointed and delayed in responding.

@lpicanco 
Thank you for the fix, it helped.

It will be released as v0.1.3.
